### PR TITLE
Add pre-commit hook instructions to contribution documentation guide

### PR DESCRIPTION
**Pull request by algomaster99**
Add instructions to setup pre-commit hook as added in [this](https://github.com/iterative/dvc.org/pull/324) pull request.

### CODE REVIEW

File: static/docs/user-guide/contributing-documentation.md [hunk 1]
```diff
@@ -48,6 +48,20 @@ follow the steps below:
...
+  style checkers on pre-commit. On Mac OS, use `brew` to install the lastest
+  version of python.
+- We **strongly** recommend initializing a [virtual
+  environment](https://virtualenv.pypa.io/en/latest/userguide/) before
+  installing the required libraries for style checkers. Follow the instructions
+  to create one:
+  ```dvc
+    $ cd dvc.org
+    $ virtualenv --python python3 .env
+    $ source .env/bin/activate
```
Comment (shcheklein) [inline static/docs/user-guide/contributing-documentation.md]: _minor_: 4 spaces inside the code blocks

**Comment by algomaster99** [comment]
@shcheklein Currently, I have removed manual indentation in code blocks since [pull request](https://github.com/iterative/dvc.org/pull/330) has been merged. I had to do it manually because prettier was unable to format it. I have opened an [issue](https://github.com/prettier/prettier/issues/6117) for it in their repository.

File: static/docs/user-guide/contributing-documentation.md [hunk 2]
```diff
@@ -43,11 +43,25 @@ follow the steps below:
...
   installed.
 - Install the dependencies by running the command `npm install`.
+- Make sure you have python 3.6 or higher installed. It will be required to run
+  style checkers on pre-commit. On Mac OS, use `brew` to install the lastest
+  version of python.
+- We **strongly** recommend initializing a [virtual
+  environment](https://virtualenv.pypa.io/en/latest/userguide/) before
+  installing the required libraries for style checkers. Follow the instructions
+  to create one:
+  ```dvc
```
Comment (shcheklein) [inline static/docs/user-guide/contributing-documentation.md]: minor: let's add an extra line between the code block and the paragraph

**Comment by shcheklein** [comment]
Let's wait on your investigation of the second PR before we do this merge. It might change a little bit.

**Comment by algomaster99** [comment]
@shcheklein yeah let's wait for prettier to respond on this issue. :sweat_smile:

**Comment by shcheklein** [comment]
@algomaster99 please review it again after I merged your prettier PR, resolve the conflict and let's merge it again.

**Comment by algomaster99** [comment]
@shcheklein Please see this. I think it is ready for merge. If there are more changes, let me know.

**Comment by shcheklein** [comment]
thanks @algomaster99 🎉

**Pull request merged**